examples/quickstart/quickstart.ipynb

```diff
@@ -550,8 +550,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "mnist_eval_step = mnist_run.steps[3]\n",
-    "fashion_mnist_eval_step = fashion_mnist_run.steps[3]"
+    "mnist_eval_step = mnist_run.steps[2]\n",
```

Comment:
We can replace this with the `get_step(name='evaluator')` paradigm to make it super clear.